Resources/Prototypes/_Harmony/Traits/disabilities.yml

```
@@ -8,3 +8,11 @@
       - BorgChassis
   components:
     - type: Hypophonia
+
+- type: trait
+  id: Uncloneable
+  name: trait-uncloneable-name
+  description: trait-uncloneable-desc
+  category: Disabilities
+  components:
+  - type: Uncloneable
```

Files should always have one empty newline at the end
```suggestion
  - type: Uncloneable

```